frappe/email/doctype/newsletter/newsletter.py

```diff
@@ -250,3 +263,11 @@ def get_newsletter_list(doctype, txt, filters, limit_start, limit_page_length=20
 			'''.format(','.join(['%s'] * len(email_group_list)),
 					limit_page_length, limit_start), email_group_list, as_dict=1)
 
+def send_scheduled_email():
```

Comment:
how is this sent without a checkbox? it means there is no control over sending!

Very bad design!